### PR TITLE
Updated disk free health checker to look for other mounts.

### PR DESCRIPTION
**Pull request by carlosroman**

### CODE REVIEW

**Comment by martingartonft** [comment]
lgtm. Nice change.

File: disk.go [hunk 1]
```diff
@@ -12,25 +13,37 @@ type diskFreeChecker struct {
...
-		PanicGuide:       "Please refer to technical summary",
-		Severity:         2,
-		TechnicalSummary: "Please clear some disk space on the 'vol' mount",
-		Checker:          dff.mountedDiskSpaceCheck,
+	var fthealthChecks = []fthealth.Check{}
+	for _, mount := range mounts.Mounts {
+		if strings.HasPrefix(mount.Device, "/dev") {
+			fthealthChecks = append(fthealthChecks, fthealth.Check{
+				BusinessImpact:   "A part of the publishing workflow might be affected",
+				Name:             fmt.Sprint("Persistent disk space check mounted on '%s' (always true for stateless nodes)", mount.MountPoint),
```
Comment (Zapadlo) [inline disk.go]: 1. No longer "(always true for stateless nodes)"
2. `'%s'` is never evaluated: `Persistent disk space check mounted on '%s' (always true for stateless nodes)/`

**Comment by kovacshuni** [comment]
+1

**Comment by Zapadlo** [comment]
+1

**Comment by david-smith-ft** [comment]
Closing due to age.

**Pull request closed without merging**